core/views.py

```diff
@@ -69,6 +71,11 @@ class PostCreateView(LoginRequiredMixin, CreateView):
     template_name = "core/post-create.html"
     login_url = "login"
 
+    def dispatch(
```

Comment:
Is this needed? Seems not doing anything.

Reply:
ok deleted